modules/assim.batch/R/pda.define.llik.R

```diff
@@ -67,18 +67,21 @@ pda.calc.error <-function(settings, con, model_out, run.id, inputs, bias.terms){
   for (k in seq_len(n.input)) {
     
     if (settings$assim.batch$inputs[[k]]$likelihood == "Laplace") {
```

Comment:
More general design question (not something that needs to be done immediately): if we defined the llik functions more generically as actual functions with a name and I/O pattern (as we do in other places), could we avoid having all these special cases? I worry this function is going to get too long and hard to maintain as the number of likelihoods increases. Plus, using functions would make it easier for users to add new functions, including doing so on the fly (e.g. in local scripts rather than recompiling this package)